requirements.txt

```diff
@@ -44,6 +44,10 @@ git+https://github.com/CenterForOpenScience/modular-odm.git@develop
 # Python markdown extensions for comment emails
 git+git://github.com/aleray/mdx_del_ins.git
 
+# Kombu with the ability to specify queue priority
```

Comment:
I'm sure you tested this but kombu, celery, and rabbitmq tend to have pretty tight version binding. I've upgraded rabbitmq before which broke celery, then upgraded celery (which didn't specify a dependency on a specific version of kombu) which then broke the upgraded celery forcing me to manually upgrade kombu. That was a fun four hours I'll never get back.